### ISSUE TITLE
`bash-language-server` crashes after update

### ISSUE THREAD

**Issue by AlexanderBrevig** [helix-editor/helix#6230]
### Summary

Using latest from branch Helix and latest from AUR `bash-language-server` `4.8.2` I get no LSP for bash.

I am not sure if this is an issue with Helix or the LSP, but maybe we could be more graceful about the issue?

Let me know if this is somehow better suited as an issue against the upstream LSP.

### Reproduction Steps

 

I tried this: Updated bash-language-server to latest version

I expected this to happen: Still get LSP for bash

Instead, this happened: No LSP for bash, since it crashes

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
2023-03-09T11:11:21.175 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","result":[null],"id":1}
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "/usr/lib/bash-language-server/node_modules/vscode-jsonrpc/lib/common/connection.js:531\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "                        responsePromise.reject(new messages_1.ResponseError(error.code, error.message, error.data));\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "                                               ^\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "ResponseError: Method not found: client/registerCapability\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "    at handleResponse (/usr/lib/bash-language-server/node_modules/vscode-jsonrpc/lib/common/connection.js:531:48)\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "    at processMessageQueue (/usr/lib/bash-language-server/node_modules/vscode-jsonrpc/lib/common/connection.js:327:17)\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "    at Immediate.<anonymous> (/usr/lib/bash-language-server/node_modules/vscode-jsonrpc/lib/common/connection.js:311:13)\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "    at processImmediate (node:internal/timers:466:21) {\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "  code: -32601,\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "  data: undefined\n"
2023-03-09T11:11:21.178 helix_lsp::transport [ERROR] err <- "}\n"
2023-03-09T11:11:21.220 mio::poll [TRACE] deregistering event source from poller
2023-03-09T11:11:21.220 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: None, method: "exit", params: None })))
2023-03-09T11:11:21.220 helix_lsp::transport [ERROR] err: <- StreamClosed
2023-03-09T11:11:21.220 mio::poll [TRACE] deregistering event source from poller
2023-03-09T11:11:21.220 helix_view::editor [DEBUG] editor status: Language server exited

```

</details>

### Platform

Linux 6.2.2-arch1-1

### Terminal Emulator

kitty 0.27.1

### Helix Version

22.05-1430-g2cf4ce23

**Comment by the-mikedavis** [comment]
This might be the same issue as pyright had (https://github.com/microsoft/pyright/pull/4666) since they both use the vscode-languageserver dependency. I can look at submitting the same patch to bash-language-server.

**Comment by the-mikedavis** [comment]
Should be fixed by https://github.com/bash-lsp/bash-language-server/pull/763

**Comment by AlexanderBrevig** [comment]
Amazing!

**Comment by AlexanderBrevig** [comment]
Should I close now or wait until the patch is available?

**Comment by the-mikedavis** [comment]
Looks like that was just merged and it should be released with the server version 4.8.3 so I'll close this out
